tests/integration/modules/test_git.py

```diff
@@ -56,7 +56,7 @@ def _worktrees_supported():
     Check if the git version is 2.5.0 or later
     '''
     try:
-        return _git_version() >= LooseVersion('2.5.0').encode()
+        return _git_version() >= LooseVersion('2.5.0')
```

Comment:
A git history shows that this `.encode()` was added for Py3 compat. Does this change regress Py3 support?